war/src/main/js/components/row-selection-controller/index.js

```diff
@@ -10,7 +10,7 @@ rowSelectionControllers.forEach(headerCheckbox => {
 
   if (tableCheckboxes.length === 0) {
     headerCheckbox.disabled = true;
-    moreOptionsButton.disabled = true;
+    moreOptionsButton?.disabled = true;
```

Comment:
Optional chaining with assignments would be a dream 😢

Reply:
```
 Invalid left-hand side in assignment expression. (13:4)
```

Reply:
Dang, didn't realise you couldn't do that. I've changed it - cheers :)